packages/interaction-tracking/src/InteractionTrackingSubscriptions.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {Interaction, Subscriber} from './InteractionTracking';
+
+import {enableInteractionTracking} from 'shared/ReactFeatureFlags';
+import {__subscriberRef} from 'interaction-tracking';
+
+let subscribers: Set<Subscriber> = (null: any);
+if (enableInteractionTracking) {
+  subscribers = new Set();
+}
+
+export function subscribe(subscriber: Subscriber): void {
+  subscribers.add(subscriber);
```

Comment:
Shouldn't these be noops if `enableInteractionTracking` is false?

Reply:
Yikes. Yup